docs/index.rst

```diff
@@ -578,7 +578,7 @@ End clients (non-CP clients):
    .. sourcecode:: bash
 
       # First get the KAFKA_CLUSTER_ID
-      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
+     KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v3/clusters | jq ".data[0].cluster_id"")
```

Comment:
IIUC this won't work due to KREST-19?

Reply:
@rnpridgeon can you please revert so we can merge?

Reply:
alternatively you could use the kafka rest credentials, for now its reverted.